.github/workflows/api2teams_ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Api2teams CI
+
+on:
+  push:
+    paths:
+      - "packages/api2teams/**"
+    branches: [main, dev, prerelease]
+  pull_request:
+    paths:
+      - "packages/api2teams/**"
+    branches: [main, dev, prerelease, api2teams]
```

Comment:
is "api2teams" a feature branch?

Reply:
Yes, this package is available only in api2teams branch, and we still need to decide whether we will merge it into dev.